scripts/cbsa/parse-metro-areas.ts

```diff
@@ -0,0 +1,92 @@
+import fs from 'fs';
+import path from 'path';
+import { groupBy, values } from 'lodash';
+const promisify = require('util').promisify;
+const csvParse = promisify(require('csv-parse'));
+
+/**
+ * This is a one-off script to parse and format metro areas from the
+ * mar2020.csv file.
+ */
+
+enum StatisticalAreaType {
+  Metro = 'Metropolitan Statistical Area',
+  Micro = 'Micropolitan Statistical Area',
+}
+
+interface CsvRow {
+  cbsaCode: string;
+  metropolitanDivisionCode: string;
+  csaCode: string;
+  cbsaTitle: string;
+  statisticalAreaType: StatisticalAreaType;
+  metropolitanDivisionTitle: string;
+  csaTitle: string;
+  countyEquivalent: string;
+  stateName: string;
+  fipsStateCode: string;
+  fipsCountyCode: string;
+  centralOutlyingCounty: string;
+}
+
+async function main() {
+  const csvText = fs.readFileSync(path.join(__dirname, 'mar2020.csv'));
+  const [, ...rows] = await csvParse(csvText);
+  const items: CsvRow[] = rows
+    .map(parseRow)
+    .filter((item: CsvRow) => item.csaCode);
```

Comment:
Am I reading this code right that we are using only CBSAs that are within a Combined Statistical Area?  

Can you add a comment or two to this code to make the business logic clearer?  

And is this logic covered by the PRD?  I didn't realize we weren't using all CBSAs, and I don't understand CSAs and CBSAs well enough to understand why we aren't using all CBSAs or just using CSAs directly.  🤔

Reply:
I updated the PR description to clarify. We actually need all the Metro Areas, not CSAs, updating the script!